# A die is rolled 24 times

# Use the CLT to estimate the probability that:

## a) the sum is greater than 84

## b) the sum is equal to 84

_______

# $S_{24} = X_{1}+X_{2}+...+X_{24}$

# $E(X_{i}) = \frac{7}{2} \implies E(S_{24}) = 24 \cdot \frac{7}{2} = 84$

# $E(X_{i}^{2}) = \frac{1}{6}(1^{2} + 2^{2} + ... + 6^{2}) = \frac{91}{6}$

# $V(X_{i}) = E(X_{i}^{2}) - E(X_{i})^{2} = \frac{35}{12}$

# $\implies V(S_{24}) = 24\cdot \frac{35}{12} = 70 \implies \sigma_{S_{24}} = \sqrt{70}$

___

# a)

# 84 is the mean of the distribution, so we integrate $\phi(x)$ from $\frac{84.5-84}{\sqrt{70}}$ to $\infty$

In [7]:
val = (84.5-84)/(70**0.5)
1-norm.cdf(val)

0.47617282233682967

# Therefore $P(S_{24}>84) = 0.4762$

______

# b)

# This time, we integrate from $-\left (\frac{84.5-84}{\sqrt{70}}\right )$ to $\left (\frac{84.5-84}{\sqrt{70}}\right )$

In [8]:
norm.cdf(val) - norm.cdf(-val)

0.047654355326340714

# Therefore $P(S_{24}=84) = 0.04765$